website/content/userguide/rsync-backups.markdown

```diff
@@ -77,8 +77,12 @@ dont_namespace = true
 EOT
 ```
 
+(Note that `scan2drive.lan` is the name of my gokrazy instance. I added the IP of the instance and defined a name in `/etc/hosts`, i.e `192.168.10.2 scan2drive.lan`)
```

Comment:
No, that’s not needed in my network. The hostnames that devices send to my router when they configure an IP address using DHCP is available for name resolution using DNS. This is a pretty common setup for home networks, but not all routers support it (some require extra settings).

I think we should just adjust this example to use `backedup` as the hostname so that it matches the `config.json` above, and no extra explanation — the concept of a hostname should be clear to gokrazy users.